CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Beadledom Changelog
```

Comment:
The beadledom-client projects are deprecated, should those be removed in the 4.0 release, or at a later date?

Reply:
Probably later.